src/app/my-messages/community-invitation-message-card/community-invitation-message-card.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { TodoNotificationComponent } from '../todo-notification/todo-notification.component';
+import { CommunityRegistrationService } from '../../shared/community-registration.service';
+import { MatDialog } from '@angular/material';
+
+@Component({
+  selector: 'app-community-invitation-message-card',
+  templateUrl: './community-invitation-message-card.component.html',
+  styleUrls: ['./community-invitation-message-card.component.scss', '../my-messages.component.scss']
```

Comment:
We could use `@import` directive to import `my-messages.component.scss` straight into `community-invitation-message-card.component.scss`.